### PR TITLE
FIX-#5477: Fix TypeError: read_sas() takes 1 positional argument but 2 were given

### LINKED ISSUE #5477: BUG: TypeError: read_sas() takes 1 positional argument but 2 were given

### PR DESCRIPTION
**Pull request by anstadnik**
FactoryDispatcher.read_sas expects only kwargs
https://github.com/modin-project/modin/blob/3e314d8686293415450e5d382547262f49aa38d8/modin/core/execution/dispatching/factories/dispatcher.py#L239-L242

## What do these changes do?

- [x] first commit message and PR title follow format outlined [here](https://modin.readthedocs.io/en/latest/development/contributing.html#commit-message-formatting)
  > **_NOTE:_**  If you edit the PR title to match this format, you need to add another commit (even if it's empty) or amend your last commit for the CI job that checks the PR title to pick up the new PR title.
- [x] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] signed commit with `git commit -s` 
- [x] Resolves #5477 
- [x] tests added and passing
- [x] module layout described at `docs/development/architecture.rst` is up-to-date

### CODE REVIEW

**Comment by anmyachev** [comment]
@anstadnik thanks for the contribution! Could you also add tests? (in `modin\pandas\test\test_io.py`)

**Comment by anstadnik** [comment]
Uf, unfortunately, I don't have the capacity.
Also, I see that my PR doesn't fulfill all the requirements.
Please, feel free to close the PR if those are crucial :)

**Comment by anmyachev** [comment]
> Uf, unfortunately, I don't have the capacity. Also, I see that my PR doesn't fulfill all the requirements. Please, feel free to close the PR if those are crucial :)

No problem, I can continue your pull request if you don't mind :)

**Comment by anstadnik** [comment]
That'd be great!

**Review by vnlitvinov — APPROVED** [review]
LGTM, thanks @anstadnik and @anmyachev!

File: modin/pandas/test/test_io.py [hunk 1]
```diff
@@ -2430,6 +2430,15 @@ def test_to_stata(self):
         )
 
 
+class TestSas:
+    def test_read_sas(self):
+        eval_io(
+            fn_name="read_sas",
+            # read_stata kwargs
```
Comment (YarShev) [inline modin/pandas/test/test_io.py:2437]: read_stata -> read_sas

**Pull request merged**